assets/component-menu-drawer.css

```diff
@@ -215,11 +215,27 @@ details[open].menu-opening > .menu-drawer__submenu {
 }
 
 .menu-drawer__utility-links {
-  padding: 2rem;
+  padding: 0;
   background-color: rgba(var(--color-foreground), 0.03);
   position: relative;
 }
 
+.header.header--has-social .menu-drawer__utility-links {
+  padding: 2rem;
+}
+
+@media screen and (max-width: 749px) {
+  .header.header--has-account:where(:not(.header--has-social):not(.header--has-localizations)) .menu-drawer__utility-links {
```

Comment:
Are we just using :where() to reduce specificity here? If so, I suppose that's reasonable. Otherwise it shouldn't be necessary.

Reply:
Yeah, I forgot to mention it, without :where it stays with the same specificity.